boring-sys/patches/boring-pq.patch

```diff
@@ -3997,6 +4410,14 @@ index cafae9d17..074ac5906 100644
 +                              const struct KYBER768_public_key *in_pub,
 +                              const uint8_t in[KYBER_ENCAP_BYTES]);
 +
++// IPDWING_encap is a deterministic function the generates and encrypts a random
++// session key from the given entropy, writing those values to |out_shared_key|
++// and |out_ciphertext|, respectively.
++OPENSSL_EXPORT void IPDWING_encap(uint8_t out_ciphertext[IPDWING_CIPHERTEXT_BYTES],
++                              uint8_t out_shared_key[IPDWING_KEY_BYTES],
```

Comment:
Unless boringSSL's variable naming conventions dictate otehrwise, I'd go with "shared_secret" here in order to align with the draft.